src/lib.rs

```diff
@@ -220,7 +325,7 @@ where
 }
 
 impl<I2C, E> Accelerometer for Lis3dh<I2C>
-    where 
+where
```

Comment:
norm calculation needs work I think. I not sure why its shifting by 4 at all?
We need to fix up sensitivity for all the different modes
shifting right divides by 2^n so better to do it that way than a floating?
>>1 /2
>>2 /4
>>3 /8
>>4 /16
>>5 /32
>>6 /64

then accel_norm wants in G units so mg * 1000=g
From datasheet
// 2g high res  1mg/digit	>>0 * 1000
// 2g normal    4mg/digit	>>2 * 1000
// 2g low       16mg/digit	>>4 * 1000

// 4g high res  2mg/digit	>>1 * 1000
// 4g normal    8mg/digit	>>3 * 1000
// 4g low       32mg/digit	>>5 * 1000

// 8g high res  4mg/digit	>>2 * 1000
// 8g normal    16mg/digit	>>4 * 1000
// 8g low       64mg/digit	>>6 * 1000

12 48, 192 dont have a cheat
// 16g high res  12mg/digit	 * 1000/12
// 16g normal    48mg/digit	 * 1000/48
// 16g low       192mg/digit	 * 1000/192